src/Processors/StepNameProcessor.cs

```diff
@@ -43,16 +45,19 @@ public StepNameResponse Process(StepNameRequest request)
             var stepText = _stepRegistry.GetStepText(parsedStepText);
             var hasAlias = _stepRegistry.HasAlias(stepText);
             var info = _stepRegistry.MethodFor(parsedStepText);
-
+            response.IsExternal = info.IsExternal;
             response.HasAlias = hasAlias;
-            response.FileName = info.FileName;
-            response.Span = new Span
+            if (!response.IsExternal)
```

Comment:
Please add test for processing external step name request.

Reply:
@BugDiver - do you want to take a shot at this?

Reply:
There are integration tests covering this scenario. I'll try to add few units as well.
And just now I realize I may have missed one case (StepPositions can be null for external reference), I'll adress that as well.